client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -21,6 +21,40 @@ import DomainPickerButton from '../domain-picker-button';
 import { selectorDebounce } from '../../constants';
 import SignupForm from '../../components/signup-form';
 
+import wp from '../../../../lib/wp';
```

Comment:
> `../../../../lib/wp`

@Automattic/team-calypso @sgomes  might be good candidate for `/packages` and optimizations. :-))

Reply:
That's a discussion that's come up a few times, and the general consensus has been that `lib/wp` needs way more than just some optimisations 🙂 

That said, an intermediate step of extracting it into a package might be worth considering? What do you think, @Automattic/team-calypso?